src/Provision.js

```diff
@@ -0,0 +1,313 @@
+/* eslint-disable no-console */
+
+const liveTypes = [
```

Comment:
we can actually get rid of these specs for live types, those are contained in asset manager now.

Reply:
So get rid of this code as well?
```
  for (let i = 0; i < liveTypes.length; i++) {
    const typeId = await client.CreateContentType({
      name: `${tenantName} - ${liveTypes[i].name}`,
      metadata: {
        ...typeMetadata,
        public: {
          ...typeMetadata.public,
          title_configuration: liveTypes[i].spec
        }
      }
    });

    await SetObjectPermissions(client, typeId, tenantAdminGroupAddress, contentAdminGroupAddress, contentUserGroupAddress);

    if (debug) {
      console.log(`\t${tenantName} - ${liveTypes[i].name}: ${typeId}`);
    }
  }
  ```

Reply:
We decided just to leave it there